astrapy/utils.py

```diff
@@ -31,9 +32,13 @@ def make_request(
         timeout=DEFAULT_TIMEOUT,
         headers={auth_header: token},
     )
+
     try:
-        return r.json()
+        if return_type == "json":
+            return r.json()
+        else:
+            return r
     except Exception as e:
-        logger.error(e)
+        logger.info(e)
```

Comment:
Maybe `warning`? Especially since we just return `None` with no record of the error

Reply:
Agreed.